index.html

```diff
@@ -48,6 +48,173 @@ <h1 class="header-block-title">
             </ul>
         </div>
     </header>
+    <div class="features">
+        <div class="features-about">
+            <h6 class="features-about__text">Your Personalized Coffee</h6>
+            <h3 class="features-about__title">COFFEE BUILD YOUR BASE</h3>
+        </div>
+
+        <div class="features-cards">
+            <div class="features-cards-row">
+                <img class="features-cards-row__img" src="./src/assets/images/img1.png" alt="">
+                <h4 class="features-cards-row__name">Nguồn gốc</h4>
+                <span class="features-cards-row__text">Những hạt cà phê Arabica, Robusta đạt chất lượng theo
+                tiêu chuẩn quốc tế.</span>
+            </div>
+
+            <div class="features-cards-row">
+                <img class="features-cards-row__img" src="./src/assets/images/img2.png" alt="">
+                <h4 class="features-cards-row__name">Chất lượng</h4>
+                <span class="features-cards-row__text">Từng một hạt cà phê là cả một quá trình tập trung cao độ của
+                nghệ nhân cà phê.</span>
+            </div>
+
+            <div class="features-cards-row">
+                <img class="features-cards-row__img" src="./src/assets/images/img3.png" alt="">
+                <h4 class="features-cards-row__name">Các loại hạt</h4>
+                <span class="features-cards-row__text">70% chất lượng tách cà phê đến từ nguồn gốc và chất
+                lượng green bean.</span>
+            </div>
+
+            <div class="features-cards-row">
+                <img class="features-cards-row__img" src="./src/assets/images/img4.png" alt="">
+                <h4 class="features-cards-row__name">Pha chế</h4>
+                <span class="features-cards-row__text">Những hạt cà phê được lột xác qua quá trình rang xay
+                kỹ lưỡng và nghiêm ngặt.</span>
+            </div>
+        </div>
+    </div>
+    <div class="slider">
+        <div class="slider-about">
+            <h6 class="slider-about__text">Choose Your Favorite</h6>
+            <h3 class="slider-about__title">CHUẨN GU ĐÚNG VỊ</h3>
+        </div>
+        <div class="slider-container">
+            <div id=2 class="slider-products active">
+                <div class="slider-products-row">
```

Comment:
I think it will be better to use class `slider-products-card`. You don't have a row at this block

Reply:
correct